azure-pipelines-product-construction-service.yml

```diff
@@ -85,6 +87,19 @@ stages:
               Set-Content -Path $(diffFolder)/before.json -Value $before
           displayName: Snapshot configuration (before)
 
+        - task: AzureCLI@2
+          name: GetAuthInfo
+          displayName: Get auth information
```

Comment:
This could be more specific as the job does only one thing, which is fetching the token

So maybe something like `GetMaestroToken` would be better. But up to you